### ISSUE TITLE
My Skills - Filter skills by work stream

### ISSUE THREAD

**Issue by yonikid15** [GCTC-NTGC/gc-digital-talent#1481]
# Description
- Create the filter section in the skills page of the candidate profile. Skills are categorized in a 3 tier structure: Parent Category -> Category -> Skill. This section includes creating accordions that represent the Parent Categories (Technical and Transferable). The accordion content will hold the second level categories. When one of the categories are clicked it should submit its list of selected items to handler function passed into the component. ~highlight it and add it to the list of filters that will be shown underneath the closed accordion (see wireframe)~. The list of skills can be passed in as props.

QUESTION: waiting on design finalization/updates for this

**[My Skills wireframe](https://www.figma.com/proto/PrvXEDac0HVp7dF7SLWVFm/Candidate-profile-interfaces---V5.0.0?node-id=1760%3A4320&scaling=scale-down-width&page-id=0%3A1&starting-point-node-id=1726%3A3054)**

Acceptance Criteria:
- [ ] Accordion has the correct skills in button and content.
- [ ] Accordion opens with second level skills below in a list of buttons.
- [ ] Accordion closes with highlighted skills below.
- [ ] Clicking on chosen filter removes it from the filters list.

This assumes the accordion component has already been created.

**Comment by tristan-orourke** [comment]
Duplicate of #1545
